deploy_and_install_scylla_ansible_script/scylla_deployment.yml

```diff
@@ -0,0 +1,170 @@
+      when: ansible_distribution == 'Debian'
+      tags:
+       - 2_java
+    - name: Update apt cache on Debian / Ubuntu
+      apt: update_cache=true
+      when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu' #and ansible_distribution_release == 'trusty')
+      tags:
+       - 2_java
+    - name: Install Java 8 on Ubuntu14, needed for Scylla release 1.7
+      apt: name=openjdk-8-jre-headless state=latest
+      when: ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'trusty'
+      tags:
+       - 2_java
+    - name: Install Java 8 on Debian, needed for Scylla release 1.7
+      apt: name=openjdk-8-jre-headless state=latest default_release=jessie-backports
+      when: ansible_distribution == 'Debian'
+      tags:
+       - 2_java
+    - name: Select correct java version on Debian / Ubuntu14
+      alternatives: name=java path=/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+      when: ansible_distribution == 'Debian' or (ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'trusty')
+      tags:
+       - 2_java
+    - name: Download Scylla {{ release }} repo for Centos 7 / RHEL 7
+      get_url: url=http://downloads.scylladb.com/rpm/centos/scylla-{{release}}.repo dest=/etc/yum.repos.d/scylla.repo
+      when: ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat'
+      tags:
+       - 3_repo
+    - name: Download Scylla {{ release }} repo for Ubuntu 14.04 (Trusty)
+      get_url: url=http://downloads.scylladb.com/deb/ubuntu/scylla-{{release}}-trusty.list dest=/etc/apt/sources.list.d/scylla.list
+      when: ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'trusty'
+      tags:
+       - 3_repo
+    - name: Download Scylla {{ release }} repo for Ubuntu 16.04 (Xenial)
+      get_url: url=http://downloads.scylladb.com/deb/ubuntu/scylla-{{release}}-xenial.list dest=/etc/apt/sources.list.d/scylla.list
+      when: ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'xenial'
+      tags:
+       - 3_repo
+    - name: Download Scylla {{ release }} repo for Debian 8 (Jessie)
+      get_url: url=http://downloads.scylladb.com.s3.amazonaws.com/deb/debian/scylla-1.7-jessie.list dest=/etc/apt/sources.list.d/scylla.list
+      when: ansible_distribution == 'Debian'
+      tags:
+       - 3_repo
+    - name: Install scylla {{ release }} on Debian / Ubuntu 
+      package: name=scylla state=present force=yes
+      when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
+      tags:
+       - 4_install
+    - name: Install scylla {{ release }} on CentOS / RHEL
+      package: name=scylla state=present
+      when: ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat'
+      tags:
+       - 4_install
+    - name: Configure Cluster name in yaml file
+      shell: sed -i -- 's/Test Cluster/{{ cluster_name }}/g' /etc/scylla/scylla.yaml
```

Comment:
Couldn't make it work and change as I wanted using lineinfile / replace (spent few good hours trying), so kept it as 'sed' command. Any suggestions are welcomed.